src/new-client/src/App/LiveRates/styled.tsx

```diff
@@ -153,3 +153,11 @@ export const DropdownOption = styled.div<{ selected: boolean }>`
     padding-left: 20px;
   }
 `
+
+export const HeaderAction = styled.button`
```

Comment:
This is duplicated across components, a good candidate to move out into a generic component